test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -423,21 +422,6 @@
           "lib/netstandard2.0/Azure.Storage.Common.dll": {}
         }
       },
-      "DotNetTI.BreakingChangeAnalysis/1.1.0-preview": {
-        "dependencies": {
-          "Marklio.Metadata": "1.2.20-beta",
```

Comment:
@brettsam  - please do look at the changes in this file. Had to update as the test caught the changes

Reply:
Added comments about this above.

Reply:
Thanks! Just saw this. Addressing in the next iteration.

Reply:
Moving `protobuf-net` from `"3.0.0-alpha.122` to `3.0.0` .  Let me know if you have concerns with this